### ISSUE TITLE
Properties that start with an underscore are displayed without their underscore

### ISSUE THREAD

**Issue by JohnRudolfLewis** [Swagger2Markup/swagger2markup#206]
Properties that start with an underscore are displayed without their underscore

**Desired:**
From the example below, I would want the "_etag" property to render with the underscore

**Actual:**
The definition is rendered without the underscore

```
convertSwagger2markup {
    swaggerInput file("swagger.yaml").toString()
    outputDir asciiDocOutputDir
    config = [
            'swagger2markup.pathsGroupedBy' : 'TAGS',
            'swagger2markup.tagOrderBy' : 'AS_IS',
            'swagger2markup.propertyOrderBy' : 'AS_IS',
            'swagger2markup.extensions.springRestDocs.snippetBaseUri': snippetsOutputDir.getAbsolutePath()]
}
```

```
swagger: '2.0'
info:
  version: 0.0.0
  title: Simple API
tags:
  - name: Zebras
  - name: Aardvarks
paths:
  /v1/zebras:
    get:
      description: I would like this to appear before Aardvarks
      tags:
        - Zebras
      responses:
        200:
          description: OK
          schema:
            $ref: "#/definitions/Zebra"

  /v1/aardvarks:
    get:
      description: I would like this to appear after zebras
      tags:
        - Aardvarks
      responses:
        200:
          description: OK
          schema:
            $ref: "#/definitions/Aardvark"

definitions:
  
 
  Aardvark:
    type: object
    allOf:
      - $ref: "#/definitions/Animal"
    properties:
      somePropertyOfAardvark:
        type: string
        description: this should be displayed in the definition of Aardvark but doesn't
      
  Zebra:
    type: object
    allOf:
      - $ref: "#/definitions/Animal"
    properties:
      somePropertyOfZebra:
        type: string
        description: this should be displayed in the definition of Zebra but doesn't
    required:
      - somePropertyOfZebra
  
  Animal:
    type: object
    properties:
      somePropertyOfAnimal:
        type: string
      "_etag":
        type: string
        readOnly: true
        description: This property is actually named _etag but shows up without the underscore. It also should show that it is a readOnly property.
```

**Comment by RobWin** [comment]
The problem is that currently bold AsciiDoc text looks like this: `*_etag*`
Since `_` is not escaped it does not work.
I change the markup-document-builder to render bold text as `**_etag**` so that it works.

**Comment by RobWin** [comment]
I changed the markup-document-builder. Result looks as follows:

![image](https://cloud.githubusercontent.com/assets/5815046/20175398/d07a99bc-a742-11e6-9e49-cf39f01ad088.png)

I publish  v1.1.1-SNAPSHOT soon.

**Comment by RobWin** [comment]
Fixed in v1.1.0
